### ISSUE TITLE
[Bug] 'Database is locked' error when calling mbtiles files after using disk_to_mbtiles

### ISSUE THREAD

**Issue by larsmaxfield** [larsmaxfield/mbutil_zyx#5]
The following error is thrown if I immediately access an MBTiles file created with `disk_to_mbtiles` (here when converting the MBTiles file to PMTiles):

```
2024-08-06 15:36:39,281 - root - ERROR: Exception for sqlite3 operational error,
...
  File "C:\art\disk_to_pmtiles\env\Lib\site-packages\pmtiles\convert.py", line 59, in mbtiles_to_pmtiles
    for row in cursor.execute(
               ^^^^^^^^^^^^^^^
sqlite3.OperationalError: database is locked
```

**Comment by larsmaxfield** [comment]
The problem appears to be that the database connection is not closed after connection. The fix seems to be to add `con.close()` at the end of `disk_to_mbtiles`.

**Comment by larsmaxfield** [comment]
Fixed with d47dc4533b57fa7c0fec92e8a8e66d564a3d2641.
